src/vs/workbench/contrib/notebook/browser/diff/diffElementViewModel.ts

```diff
@@ -226,10 +230,10 @@ export abstract class DiffElementCellViewModelBase extends DiffElementViewModelB
 			editorMargin: 0,
 			metadataHeight: 0,
 			cellStatusHeight,
-			metadataStatusHeight: 25,
+			metadataStatusHeight: this.ignoreMetadata ? 0 : 25,
```

Comment:
@rebornix 
Simple fix, but easily reproducible when testing show/hide metadata & notebook metadata
Hence part of #208280